Gopkg.toml

```diff
@@ -0,0 +1,70 @@
+# Gopkg.toml example
+#
+# Refer to https://golang.github.io/dep/docs/Gopkg.toml.html
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
+
+
+[[constraint]]
+  name = "github.com/MysteriumNetwork/payments"
+  version = "0.0.5-beta"
+
+[[constraint]]
+  name = "github.com/chzyer/readline"
+  version = "1.4.0"
+
+[[constraint]]
+  name = "github.com/cihub/seelog"
+  version = "2.6.0"
+
+[[constraint]]
+  name = "github.com/ethereum/go-ethereum"
+  version = "=1.8.14"
+
+[[constraint]]
+  name = "github.com/julienschmidt/httprouter"
+  version = "1.1.0"
+
+[[constraint]]
+  name = "github.com/nats-io/go-nats"
+  version = "1.4.0"
+
+[[constraint]]
+  name = "github.com/oschwald/geoip2-golang"
+  version = "1.1.0"
+
+[[constraint]]
+  name = "github.com/satori/go.uuid"
+  version = "1.2.0"
+
+[[constraint]]
+  name = "github.com/stretchr/testify"
+  version = "1.2.2"
+
+[prune]
+  go-tests = true
+  unused-packages = true
+
+  [[prune.project]]
```

Comment:
Global unused-packages is a little evil I agree - so no we can specificy this flag per dependency? It would be a bit tricky to track if we need this flag for specific depedency or not - especially if it has transitive dependencies. and this flag is very important if package uses cgo - as any .c .h files will be pruned if this flag is true. That's a bit of consideration

Reply:
Not sure when it was introduced. 
Found the only bug when it was broken: https://github.com/golang/dep/issues/1561 )